src/main/resources/db/migration/courtcase/V2176__add_audit_column_for_hearing_outcome_fk_hearing_defendant_id.sql

```diff
@@ -0,0 +1,9 @@
+BEGIN;
+
+ALTER TABLE if exists hearing_outcome_aud ADD COLUMN fk_hearing_defendant_id integer;
```

Comment:
Hi, for my own curiosity, what is `if exists` checking ? is it checking that the table `hearing_outcome_aud` exists or the column `fk_hearing_defendant_id`?

If it's checking the `fk_hearing_defendant_id` then maybe something like

`ALTER TABLE hearing_outcome_aud ADD COLUMN IF NOT EXISTS fk_hearing_defendant_id integer;`